mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml

```diff
@@ -235,6 +251,9 @@
     <string name="fxa_preboarding_primary_button">Sign in</string>
     <!-- Text on the cancel button for the Firefox Account sign-in pre-boarding, where we introduce the send tab feature. -->
     <string name="fxa_preboarding_secondary_button">Not now</string>
+    <!-- Shown on the onboarding page that explains how to receive tabs that are sent from other devices.
+         This text is announced to screen reader users when they focus an image that will display a graphic explaining the receive tab process-->
+    <string name="fxa_preboarding_image_a11y">Illustration of tabs sent from phone and tablet to Fire TV</string>
```

Comment:
We usually use "%1$s" for the app name (and explain in comments what "%1$s" is going to be replaced with). However I see that there are still a few occurrences of the app name in current strings. I won't block on this, but would be good one day to go in and update all of those.
(CC @dnarcese so she is aware :) )